x/data/features/msg_anchor.feature

```diff
@@ -0,0 +1,54 @@
+Feature: MsgAnchor
+
+  Rule: only a valid sender is accepted
```

Comment:
If rule is to be useful it should describe what valid means, sender must be a non-empty bech32 address

Reply:
I can update to be more specific. The BDD book uses "only valid address is accepted" as an example.

Reply:
Interesting